src/WhiteLibrary/__init__.py

```diff
@@ -146,7 +151,7 @@ def __init__(self):
                           TreeKeywords(self),
                           UiItemKeywords(self),
                           WindowKeywords(self),
-                          ScreenshotKeywords(self)]
+                          ScreenshotKeywords(self, screenshot_dir)]
```

Comment:
screenshotkeywords takes 2 parameters, state & screenshot_dir but only path is pased here ..

Reply:
"self" (the instance of WhiteLibrary) is given as the state parameter, state is used similarly to ctx in SeleniumLibrary